app/uk/gov/hmrc/helptosavefrontend/controllers/IntroductionController.scala

```diff
@@ -66,4 +66,8 @@ class IntroductionController @Inject() (val authConnector: AuthConnector,
   def showPrivacyPage: Action[AnyContent] = unprotected { implicit request ⇒ implicit htsContext ⇒
     Ok(views.html.core.privacy())
   }
+
+  def getAboutPage: Action[AnyContent] = unprotected { implicit request ⇒ implicit htsContect ⇒
```

Comment:
is this page only supposed to be accessible to people who have an account? If that were the case we'd want to make sure someone actually has an account here before displaying the content rather than use `unprotected` here which opens up the content to everyone

Reply:
if we need to change this from `unprotected` this would be a BE change